<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficMonitorWidget</class>
 <widget class="QWidget" name="TrafficMonitorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>139</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="horizontalSpacing">
    <number>3</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblHost">
     <property name="text">
      <string>Host</string>
     </property>
     <property name="buddy">
      <cstring>cmbHost</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="cmbHost">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinPort">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lblTimeout">
     <property name="text">
      <string>Timeout</string>
     </property>
     <property name="buddy">
      <cstring>spinTimeout</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="spinTimeout">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblProfile">
     <property name="text">
      <string>Profile</string>
     </property>
     <property name="buddy">
      <cstring>cmbProfile</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbProfile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnSnmpCredentials">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblRetries">
     <property name="text">
      <string>Retries</string>
     </property>
     <property name="buddy">
      <cstring>spinRetries</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinRetries">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblCheck">
     <property name="text">
      <string>Check</string>
     </property>
     <property name="buddy">
      <cstring>editNeworkInterface</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="editNeworkInterface"/>
     </item>
     <item>
      <widget class="QToolButton" name="btnNetworkInterface">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/browse.png</normaloff>:/img/browse.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblAlertIf">
     <property name="text">
      <string>Alert if</string>
     </property>
     <property name="buddy">
      <cstring>cmbAlertSelect</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cmbAlertSelect">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <item>
        <property name="text">
         <string>In/Out traffic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Income traffic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Outgoing traffic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In Discards</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>In Errors</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Out Discards</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Out Errors</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Output queue</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAlertSigns">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtCurrent</enum>
       </property>
       <item>
        <property name="text">
         <string>&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinAlertValue">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAlertPkt">
       <item>
        <property name="text">
         <string>pkt</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAlertKbMb">
       <item>
        <property name="text">
         <string>KB/sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>KB/min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MB/sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MB/min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kbits/sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kbits/min</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mbits/sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mbits/min</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbAlertPktT">
       <item>
        <property name="text">
         <string>pkt/sec</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pkt/min</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
